homeassistant/components/knx/schema.py

```diff
@@ -400,14 +422,11 @@ class LightSchema:
             }
         ),
         vol.Any(
-            # either global "address" or all addresses for individual colors are required
+            # either global "address" "individual_colors" is required
```

Comment:
```suggestion
            # either global "address" or "individual_colors" is required
```
Oh, the "or" must have been fallen under the table. Thanks!